scenario06/README.md

```diff
@@ -0,0 +1,123 @@
+# Scenario 6: Deploy cilium, spire and k8s worload registrar
+
+Description of the scenario: the goal of the scenario exposed by the image below is to use the SPIRE [k8s-workload-registrar](https://github.com/spiffe/spire/tree/main/support/k8s/k8s-workload-registrar) component to automatically assign the SPIFFE ID to a workload (nginx) and verify that the ID was created as a k8s label in cilium. This label can be used for policy enforcement using the [Cilium Network Policies](https://docs.cilium.io/en/stable/concepts/kubernetes/policy/#ciliumnetworkpolicy) (CNPs). 
+
+<img src="../imgs/scenario06.png" alt="drawing" width="800"/>
+
+For this tutorial the following steps will be performed:
```

Comment:
These steps numbers follows the steps numbers in the image? I was little confused about that